staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -176,6 +176,9 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.DurationVar(&s.StorageConfig.CountMetricPollPeriod, "etcd-count-metric-poll-period", s.StorageConfig.CountMetricPollPeriod, ""+
 		"Frequency of polling etcd for number of resources per type. 0 disables the metric collection.")
+
+	fs.DurationVar(&s.StorageConfig.DbMetricPollInterval, "etcd-db-metric-poll-interval", s.StorageConfig.DbMetricPollInterval,
+		"The interval of requests to poll etcd and update metrics. 0 disables the metrics collection")
```

Comment:
I am unsure neither. Apiserver already has a lot of flags. I was thinking for some users running super heavy workloads, they might want to monitor this metric more often. If so, this flag could be useful.

Reply:
FYI. Today we do have a similar flag: --etcd-count-metric-poll-period

Reply:
Agree that it could be useful, most of the time db size is pretty stable, but there are cases where overuse of etcd would cause db size increase more often than others.

Reply:
s/metrics/metric/

Reply:
> s/metrics/metric/

Done.